src/currency-field/CurrencyField.stories.ts

```diff
@@ -163,7 +164,9 @@ export const Error_Label = ErrorStory<CurrencyField, BaseArgs>('omni-currency-fi
 
 export const Value = ValueStory<CurrencyField, BaseArgs>('omni-currency-field', '1200.50');
 
-export const Clear = ClearableStory<CurrencyField, BaseArgs>('omni-currency-field', '1200.50');
+export const Clearable = ClearableStory<CurrencyField, BaseArgs>('omni-currency-field', '1200.50');
+
+export const Custom_Clear_Slot_Icon = CustomClearableSlotIcon<CurrencyField, BaseArgs>('omni-currency-field', '1200.50');
```

Comment:
Don't think Icon needs to be part of the name here. Anything can be slotted, not necessarily an icon.

Reply:
This comment applies to all the stories using this name.